Packs/Base/Scripts/DBotFindSimilarIncidents/DBotFindSimilarIncidents.yml

```diff
@@ -0,0 +1,180 @@
+args:
+- default: false
+  description: Incident id to get prediction, if empty - using the current incident
+    id.
+  isArray: false
+  name: incidentId
+  required: false
+  secret: false
+- default: false
+  description: 'Fields with short or long text that will be taken into account in
+    order to compute similarity - Example: commandline, URL...'
+  isArray: false
+  name: similarTextField
+  required: false
+  secret: false
+- default: false
+  description: Fields for which exact match is not necessary but will boost the score.
```

Comment:
Most descriptions can be a bit clearer - 

- fields = Indicators fields/incident fields 
- decide one terminology of score/confidence/similarity, it's a bit confusing having both (seems like "confidence" is more of a "threshold")  
- Try to reorder that all indicators fields arguments will appear together